README.md

```diff
@@ -1,24 +1,54 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails Blog
 
-Things you may want to cover:
+This project is full functional blog application, that includes users, posts, comments, and likes. The project code is following the SOLID principles and other best practices.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Rails 7.0
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these simple steps.
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+- Make sure you already installed Ruby version 3 or higher and Git. 
 
-* Services (job queues, cache servers, search engines, etc.)
+### Setup
 
-* Deployment instructions
+Clone the repository using the below commands.
 
-* ...
+- `git clone git@github.com:aimalamiri/Rails-Blog.git `
+- `cd Rails-Blog`
+-  Make sure to create a database and add it's configuration into the `cofig/database.yml`
+- `rails db:migrate`
+- `rails server`
+
+### Usage
+
+Kindly modify the files as needed.
+
+## Author
+
+👤 **Aimal Aimiri**
+
+- GitHub: [@aimalamiri](https://github.com/aimalamiri)
+- Twitter: [@meaimal](https://twitter.com/meaimal)
+- LinkedIn: [Aimal Amiri](https://linkedin.com/in/aimal-amiri)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/aimalamiri/Rails-Blog/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ]  [OPTIONAL] Please don't forget to create the MIT license file as you linked in your readme

Reply:
Thanks, @Rustamxon7 I'm done with the license file.